src/randoop/JunitFileWriter.java

```diff
@@ -85,19 +87,12 @@ private void createOutputDir() {
    *
    * the default junit class name is the driver class name + index
    */
+  
```

Comment:
What is the point of this blank line?  I like to jam the comment against the method body, especially since the last line of the comment is practically a blank line already.  It's more standard -- but more importantly, it's what the rest of the codebase does, and consistency makes the code easier to read.